projects/elife.yaml

```diff
@@ -947,82 +947,41 @@ iiif:
         type: t2.medium
         ports:
             - 22
-            - 80
-            - 443
+            - 80:
+                cidr-ip: 10.0.0.0/16 # ELB only
+        ec2:
+            cluster-size: 2
+        elb:
+            protocol: 
+                - https
+            healthcheck:
+                path: /
+        cloudfront:
+            subdomains: 
+                - "{instance}--cdn-iiif"
+            headers:
+                - Host
+        ext:
+            size: 30 # GB
```

Comment:
can be tested with `./bldr update_template:iiif--end2end`. It will show the changes to the CloudFormation template (there shouldn't be any), asking for confirmation (press Ctrl-C)